tools/pythonpkg/src/pandas/analyzer.cpp

```diff
@@ -391,7 +401,8 @@ LogicalType PandasAnalyzer::InnerAnalyze(py::object column, bool &can_convert, b
 		increment = GetSampleIncrement(rows);
 	}
 	for (idx_t i = increment; i < rows; i += increment) {
-		auto next_item_type = GetItemType(row(i), can_convert);
+		auto obj = FindFirstNonNull(row, i, increment);
```

Comment:
Should we not increment `i` as well, otherwise we might sample the same value multiple times, correct?